src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RegisterChecks/RoepaoTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Controllers.Roatp.Apply;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.Validators.Roatp;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.Api.Types.Models.Validation;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.Gateway.RegisterChecks
+{
+    [TestFixture]
+    public class RoepaoTests
```

Comment:
Picky but can you rename to RegisterOf...it took me a while to work out what roepoa meant. 😁

Reply:
Could argue the same for RoATP , ROTO, UKRLP, etc... 😉